lib/ui/profile/Profile.dart

```diff
@@ -525,6 +525,8 @@ class Profile extends StatelessWidget {
       context,
       FarmDetails(
         viewModel: FarmDetailsViewModel(
+          removeProfile: viewModel.remove,
+          editProfile: () => viewModel.newAccountFlow.run(context),
```

Comment:
We should do a new chat flow to edit the profile, right now is creating a new one. 
https://amidodevelopment.atlassian.net/browse/FARM-513

Reply:
yes for sure